components/raftstore-v2/src/operation/command/mod.rs

```diff
@@ -320,11 +320,10 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
     }
 
     pub fn on_apply_res<T>(&mut self, ctx: &mut StoreContext<EK, ER, T>, apply_res: ApplyRes) {
-        if !self.serving() {
-            return;
+        if !self.serving() || !apply_res.admin_result.is_empty() {
```

Comment:
So what if it's self.serving() && apply_res.admin_result.is_empty(). Should it return immediately?

Reply:
No, it should keep processing. I returned early in the past to reduce processing, but now it seems keep processing can get more benefits.